app/assets/stylesheets/new_board/_story.scss

```diff
@@ -94,8 +94,9 @@
         }
 
         &__title {
-          word-break: break-all;
+          word-break: break-word;
           flex: 1;
+          text-align: justify;
```

Comment:
why not `text-align: left` here? `justify` might create some gaps depending on the word count and space available.

Reply:
It was a purely aesthetic choice, so it fills the space in a more organized manner. Would you suggest changing it to `left`?

Reply:
I just thought about the gaps it might create, but if you think it is more aesthetic pleasing don't worry about my comment :D